src/fabric_cli/utils/fab_cmd_set_utils.py

```diff
@@ -10,27 +10,43 @@
 from fabric_cli.utils import fab_jmespath as utils_jmespath
 
 
-def validate_item_query(query_value: str) -> None:
+def validate_item_query(query_value: str, item=None) -> None:
     """Validate that a query string is allowed for item metadata modification.
 
     Args:
         query_value: Query string to validate. Must be an allowed metadata key,
-            or start with "definition." or "properties.".
+            or start with "definition." or "properties.". Must not contain
+            filters or wildcards.
+        item: Optional Item object to validate command support for definition queries.
 
     Raises:
-        FabricCLIError: If the query is not in an allowed format.
+        FabricCLIError: If the query is not in an allowed format, contains
+            filters/wildcards, or if definition update is not supported for the item type.
     """
-    if not (
-        query_value in fab_constant.ITEM_SET_ALLOWED_METADATA_KEYS
-        or query_value == fab_constant.ITEM_QUERY_DEFINITION
-        or query_value.startswith(f"{fab_constant.ITEM_QUERY_DEFINITION}.")
-        or query_value.startswith(f"{fab_constant.ITEM_QUERY_PROPERTIES}.")
-    ):
+    allowed_keys = fab_constant.ITEM_SET_ALLOWED_METADATA_KEYS + [
```

Comment:
what about ITEM_QUERY_PROPERTIES?

Reply:
This is a part of ITEM_SET_ALLOWED_METADATA_KEYS - it's in the item's metadata